packages/fabric8-ui/src/app/space/app-launcher/services/app-launcher-mission-runtime.service.ts

```diff
@@ -36,7 +43,16 @@ export class AppLauncherMissionRuntimeService extends MissionRuntimeService {
     }
     return this.http
       .get<Catalog>(this.END_POINT + this.API_BASE, { headers: this.headers }).pipe(
-      catchError(this.handleError));
+        map((catalog: Catalog) => {
+          if (!this.isNodeBoosterEnable) {
```

Comment:
The value of `isNodeBoosterEnable` is also asynchronous. Therefore you should combine this observable with the result of the feature toggle service instead of assuming that the value of `isNodeBoosterEnable` has been asynchronous set in correct order.

Reply:
@christianvogt This is done as per @corinnekrych suggestion.